help_to_heat/frontdoor/interface.py

```diff
@@ -525,14 +525,74 @@ def parse_dpa_to_address(self, dpa):
 
 
 class EPC(Entity):
-    @with_schema(load=GetScottishEPCSchema, dump=EPCSchema)
+    @with_schema(load=GetScottishEPCSchema)
     def get_epc_scotland(self, uprn):
         try:
             epc = portal.models.ScottishEpcRating.objects.get(uprn=uprn)
         except portal.models.ScottishEpcRating.DoesNotExist:
             return {}
 
-        return {"uprn": epc.uprn, "rating": epc.rating, "date": epc.date}
+        improvements = epc.improvements
+        alternative_improvements = epc.alternative_improvements
+        all_improvements = improvements + (
+            "Alternatives: " + alternative_improvements if alternative_improvements else ""
+        )
```

Comment:
Two questions:
1. Should there be a space and/or some punctuation before 'Alternatives'?
2. Is `epc.improvements` ever going to be `None`? We'll need a fallback if so

Reply:
It's not something I'm particularly concerned for, the load from the CSV means anything empty should become an empty string. If we want to account for it, then I'd probably want to `or ""` the other fields too

Reply:
I think throwing something in before 'Alternatives' would still be a good idea, so we don't have to make assumptions about the content of `epc.improvements`

Reply:
Perhaps `|| Alternatives `? Like a separator but with more weight to it